### ISSUE TITLE
Date issues

### ISSUE THREAD

**Issue by RaniRaven** [Anchormen/sql4es#36]
I notices that a java.sql.Date is returned when date type is declared on the mapping. However, in this java version the ability of Date to return time also is deprecated.
It seems like ElasticSearch's date type should be exposed as java.sql.Timestamp

**Comment by sudhirkumar1980** [comment]
Are you able to resolve this issue?

**Comment by RaniRaven** [comment]
Yes, by changing the return type to Timestamp, although it will work for application which is not a strict 3rd party one.

**Comment by mallim** [comment]
Testing 6.4.0 and having the same issue, I have to change to timestamp to pass the test case
